osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableTaikoHitObject.cs

```diff
@@ -9,10 +9,62 @@
 using System.Linq;
 using osu.Game.Audio;
 using System.Collections.Generic;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Primitives;
 
 namespace osu.Game.Rulesets.Taiko.Objects.Drawables
 {
-    public abstract class DrawableTaikoHitObject<TaikoHitType> : DrawableHitObject<TaikoHitObject>, IKeyBindingHandler<TaikoAction>
+    public abstract class DrawableTaikoHitObject : DrawableHitObject<TaikoHitObject>, IKeyBindingHandler<TaikoAction>
+    {
+        protected readonly Container Content;
+        public readonly Container ProxiedContent;
```

Comment:
I don't think this should be exposed at all (should be `private`) to avoid accidental use of `Add`/`Remove`. We can add a `protected CreateProxiedContent` or similar for the one public usage.